apps/network-alerts/js/index.js

```diff
@@ -31,23 +47,25 @@
       return;
     }
 
-    var req = navigator.mozSettings.createLock().get(CMAS_ENABLED_KEY);
+    var sendAlertPromise =
+      isPresidentialAlert(message) ?
```

Comment:
I thought that in the future if we support CDMA in the Network Alerts app it makes more sense to pass the whole message (because CDMA will have other properties).